### DISCUSSION TITLE
Welcome to finalmq Discussions!

### DISCUSSION THREAD

**Discussion by bexoft** [bexoft/finalmq#35] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mnaveedb** [comment]
Hi All, 


SSL 3.xx is shipped with the 22.04 LTS release of ubuntu.  There is only one deprecated function used in the finalmq. 
Can we create the ticket to address this.?
